js/gate.js

```diff
@@ -68,6 +68,7 @@ module.exports = class gate extends Exchange {
                 'borrowMargin': true,
                 'cancelAllOrders': true,
                 'cancelOrder': true,
+                'createDepositAddress': 'emulated',
```

Comment:
Should we use `true` instead of `emulated`? I found we did this in bitfinex.

Reply:
@sc0Vu actually I changed my opinion, the endpoint will generate the address if does not exist so this is not emulated behavior 
![image](https://user-images.githubusercontent.com/43336371/212944133-aeaa3f2b-ada4-4fd7-8f1f-c587c2c8b7e1.png)